### PR TITLE
Rewrite Lando page for the new Lando integration.

### PR DESCRIPTION
**Pull request by Crell**
Don't merge until Mike Pirog has given it his approval.

### CODE REVIEW

**Review by otaviojava — APPROVED** [review]

File: docs/src/development/local/lando.md [hunk 1]
```diff
@@ -1,67 +1,45 @@
 ---
-title: "Lando"
+title: Using Lando for local development
+sidebarTitle: "Lando"
 weight: 3
 ---
 
-# Using Lando for local development
+[Lando](https://github.com/lando/lando) is Platform.sh's recommended local development tool.  It is a third party tool developed by [Tandem](https://thinktandem.io), which now includes direct support for Platform.sh projects.  Lando can read your Platform.sh configuration files and produce an approximately equivalent configuration using Docker with minimal effort on the part of the developer.
```
Comment (pirog) [inline docs/src/development/local/lando.md:7]: "approximately equivalent" might be underselling how close the configuration actually is

**Comment by pirog** [comment]
@Crell added a small comment but beyond that LGTM!

File: docs/src/development/local/lando.md [hunk 2]
```diff
@@ -1,67 +1,45 @@
...
 
-See the [Lando documentation](https://docs.devwithlando.io/) for installing and setting up Lando on your system.
+See the [Lando documentation](https://docs.lando.dev/config/platformsh.html) for installing and setting up Lando on your system.  A quick-start guide is included below, but the Lando documentation is the primary source of truth.
 
-Lando will ask you to create a `.lando.yml` file in your application root, which functions similarly to the `.platform.app.yaml` file.  (Note the different file extension.)  It is safe to check this file into your Git repository as Platform.sh will simply ignore it.
+## Quick start
 
-If your application is one of those with a specific "recipe" available from Lando, you can use that directly in your `.lando.yml` file.  It can be customized further as needed for your application, and some customizations are specific to certain applications.
+First, ensure that you have the Platform.sh CLI installed and that you have authenticated your account.  Then, [install Lando](https://docs.lando.dev/basics/installation.html) and Docker per Lando's installation instructions.
```
Comment (pirog) [inline docs/src/development/local/lando.md:15]: > First, ensure that you have the Platform.sh CLI installed and that you have authenticated your account. 

This is not a required step for using Lando with Platform.sh.

Comment (pirog) [inline docs/src/development/local/lando.md:15]: > Then, [install Lando](https://docs.lando.dev/basics/installation.html) and Docker per Lando's installation instructions.

In most circumstances Lando will automatically install Docker for you.

**Pull request merged**